### PR TITLE
Fixes twitter logo color

### LINKED ISSUE #1367: [UI] Hovering issues for Twitter icon
### Description

image

### Expected Behavior

The Twitter X logo should be white on hover
### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Layak-Ali**
**Description**

This PR fixes #1367 

**Notes for Reviewers**
This fixes twitter logo from blue to white on hover

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 
screenshot :
Screenshot (20)

### CODE REVIEW

**Comment by Ghat0tkach** [comment]
Hey @Layak-Ali 
Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

Make sure to attend us on today's website meeting🙌.
You'll get the meeting link on slack

File: _sass/footer.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@
       }
 
       &.twitter-icon:hover svg {
-        fill: #55acee;
+        fill: #ffffff;
```
[the remark this answers is not in the document]
Comment (Layak-Ali) [reply _sass/footer.scss:38]: @thisiskaransgit, i just opened a pull request #1379 that references the  color through rootvariables.scss. I hope i did it right. Please reply me if there are any other things i should know for a beginner.

**Comment by Layak-Ali** [comment]
@thisiskaransgit @Ghat0tkach can you merge #1379 ?. I referenced it the way you asked.

**Comment by Chadha93** [comment]
@Layak-Ali Closing out this as merged your another PR with the same fix.

**Pull request closed without merging**